ops/terraform/env/mgmt/kms-data-keys.tf

```diff
@@ -29,6 +29,30 @@ resource "aws_kms_alias" "data_keys" {
   target_key_id = aws_kms_key.data_keys[each.key].arn
 }
 
+resource "aws_kms_key" "bfd_mgmt_cmk" {
```

Comment:
Can this key and its alias be rolled up adjacent to the existing data keys? e.g. 
```hcl
resource "aws_kms_key" "data_keys" {
  for_each = toset(concat(local.established_envs, ['mgmt']))
/*..snipped..*/
}

resource "aws_kms_alias" "data_keys" {
  for_each = toset(concat(local.established_envs, ["mgmt"]))
/*..snipped..*/
}
```

I haven't read this solution closely enough to make this a suggestion, so... it really is just a question–are there sufficient differences between the `mgmt` data cmk and the other data cmks to warrant separate definitions?

Reply:
I went with this suggestion and verified that the resources were imported.